### PR TITLE
0.11.3 build 0

### LINKED ISSUE #29: Exclude MDTraj from builds
### Comment:

MDTraj is not a requirement in the source code (it's used only for testing or within `@requires_package("mdtraj")` blocks around rarely-used methods) and we should not package it as such:

```shell
$ grep -r mdtraj ../openff-toolkit/openff/toolkit
../openff-toolkit/openff/toolkit/topology/molecule.py:             such as mdtraj DSL, pymol selections, atom index slices, etc? We
../openff-toolkit/openff/toolkit/topology/topology.py:        * Do we want to generalize this to other kinds of queries too, like mdtraj DSL, pymol selections, atom index
../openff-toolkit/openff/toolkit/topology/topology.py:    @requires_package("mdtraj")
../openff-toolkit/openff/toolkit/topology/topology.py:    def from_mdtraj(cls, mdtraj_topology, unique_molecules=None):
../openff-toolkit/openff/toolkit/topology/topology.py:        mdtraj_topology : mdtraj.Topology
../openff-toolkit/openff/toolkit/topology/topology.py:            mdtraj_topology.to_openmm(), unique_molecules=unique_molecules
../openff-toolkit/openff/toolkit/topology/topology.py:    @requires_package("mdtraj")
../openff-toolkit/openff/toolkit/topology/topology.py:    def _to_mdtraj(self):
../openff-toolkit/openff/toolkit/topology/topology.py:        mdtraj_topology : mdtraj.Topology
../openff-toolkit/openff/toolkit/topology/topology.py:        import mdtraj as md
../openff-toolkit/openff/toolkit/tests/test_topology.py:    @requires_pkg("mdtraj")
../openff-toolkit/openff/toolkit/tests/test_topology.py:    def test_from_mdtraj(self):
../openff-toolkit/openff/toolkit/tests/test_topology.py:        import mdtraj as md
../openff-toolkit/openff/toolkit/tests/test_topology.py:            Topology.from_mdtraj(trj.top)
../openff-toolkit/openff/toolkit/tests/test_topology.py:        top = Topology.from_mdtraj(trj.top, unique_molecules=unique_molecules)
../openff-toolkit/openff/toolkit/data/reference_energies/reference_after_802.json.txt:mdtraj                    1.9.5            py37h22c4154_0    conda-forge
../openff-toolkit/openff/toolkit/data/reference_energies/reference_after_1007.json.txt:  - mdtraj=1.9.5=py38h711e445_1
```

As we know, it's pulled in by a lot of other packages and is probably in users' environments anyway:

```shell
$ CONDA_SUBDIR=linux-64 mamba repoquery whoneeds mdtraj -c conda-forge |
wc -l
     415

### PR DESCRIPTION
**Pull request by j-wags**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Closes #29 
* [x] Supersedes #33

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -63,6 +61,7 @@ outputs:
         - ambertools >=20
         - {{ pin_subpackage('openff-toolkit-base', exact=True) }}
         - notebook
+	- ipywidgets >=7,<8
```
Comment (mattwthompson) [inline recipe/meta.yaml:64]: Tabs vs. spaces maybe? I've never seen whitespace in an added line show up as red before
```suggestion
        - ipywidgets =7
```
This should be the same thing, a little simpler

Comment (j-wags) [reply recipe/meta.yaml]: Nice. And I gotta update my emacs config to deal with those tabs!

**Comment by j-wags** [comment]
@conda-forge-admin, please rerender

**Review by mattwthompson — APPROVED** [review]
LGTM once builds pass. I don't recall any other changes that should get pulled intot hese builds.

It'd be nice to have #33 merged in at some point

**Comment by j-wags** [comment]
Oh, yeah, I'm game to move mdtraj. Will update sha and move it our of base in the next commit.

**Pull request merged**